src/Traits/ComparisonTrait.php

```diff
@@ -240,6 +240,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
+     *
+     * @return bool
+     */
+    public function isNextWeek()
+    {
+        return $this->weekOfYear === static::now($this->tz)->addWeek()->weekOfYear;
```

Comment:
I can change it, but this test case will fail. What is the correct behavior? (please see also: https://github.com/cakephp/chronos/pull/122#discussion_r96583016)

```php
    public function testIsNextWeekTrue($class)
    {
        $this->assertTrue($class::now()->addWeek()->isNextWeek());
    }
```